bftengine/src/bftengine/Crypto.cpp

```diff
@@ -177,50 +177,43 @@ class RSAVerifierInternal {
 
 RSASigner::RSASigner(const char* privateKey) {
   StringSource s(privateKey, true, new HexDecoder);
-  d = new RSASignerInternal(s);
+  impl.reset(new Impl(s));
```

Comment:
To complete the abstraction from raw pointers, can we use make_unique() here?

Reply:
Good point, done.